src/bigstring.ml

```diff
@@ -395,7 +395,8 @@ let lines s = split ~by:'\n' s
 (** {2 Memory-map} *)
 
 let map_file_descr ?pos ?(shared=false) fd len =
-  B.map_file fd ?pos Bigarray.char Bigarray.c_layout shared len
+  Bigarray.array1_of_genarray
```

Comment:
I suppose we should move this to bigstring.unix or something like that, with an explicit dep on unix (and some jbuilder magic to stil work pre-4.06)

Reply:
I suggest to cancel this change and just ignore the warning…

Reply:
that sounds like a good idea indeed :-)
do you want to do it, or should I do it after merging?